_data/navbars/maintenance.yml

```diff
@@ -9,11 +9,17 @@ section:
     path: /maintenance/kubernetes-upgrade
   - title: OpenStack
     path: /maintenance/openstack-upgrade
-- title: Decommission a node
+- title: Monitor
+  path: /maintenance/monitor
```

Comment:
@bmckercher123 FYI you are seeing semaphoreci failures because you added this page to the sidebar without creating the actual page.

I've pushed a commit to add /maintenance/monitor.md

Reply:
ugh! Sorry about that.